src/vs/workbench/contrib/welcome/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -395,5 +396,158 @@ export const content: GettingStartedContent = [
 				}
 			]
 		}
+	},
+
+	{
+		id: 'Intermediate',
+		title: localize('gettingStarted.intermediate.title', "Boost your Productivity"),
+		icon: intermediateIcon,
+		description: localize('gettingStarted.intermediate.description', "Optimize your development workflow with these tips & tricks."),
+		content: {
+			type: 'items',
+			items: [
+				{
+					id: 'playground',
+					title: localize('gettingStarted.playground.title', "Redefine your editing skills"),
+					description: localize('gettingStarted.playground.description', "Want to code faster and smarter? Practice powerful code editing feature without creating files or a project."),
+					button: {
+						title: localize('gettingStarted.playground.button', "Interactive Playground"),
+						command: 'workbench.action.showInteractivePlayground'
+					},
+					doneOn: { commandExecuted: 'workbench.action.showInteractivePlayground' },
+					media: {
+						type: 'image', altText: 'Interactive Playground.', path: {
+							dark: 'dark/playground.png',
+							light: 'light/playground.png',
+							hc: 'light/playground.png'
+						},
+					},
+				},
+				{
+					id: 'splitview',
+					title: localize('gettingStarted.splitview.title', "Side by side editing"),
+					description: localize('gettingStarted.splitview.description', "Make the most of your screen estate by opening files side by side vertically and horizontally."),
+					button: {
+						title: localize('gettingStarted.splitview.button', "Split Editor"),
+						command: 'workbench.action.splitEditor'
+					},
+					doneOn: { commandExecuted: 'workbench.action.splitEditor' },
+					media: {
+						type: 'image', altText: 'Multiple editors in split view.', path: {
+							dark: 'dark/splitview.png',
+							light: 'light/splitview.png',
+							hc: 'light/splitview.png'
+						},
+					},
+				},
+				{
+					id: 'debugging',
+					title: localize('gettingStarted.debug.title', "Run, test & debug your code"),
+					description: localize('gettingStarted.debug.description', "Catch errors faster with an accelerated edit, build and debug loop."),
+					when: 'workspaceFolderCount != 0',
+					button: {
+						title: localize('gettingStarted.debug.button', "Run your Project"),
+						command: 'workbench.action.debug.selectandstart'
+					},
+					doneOn: { commandExecuted: 'workbench.action.debug.selectandstart' },
+					media: {
+						type: 'image', altText: 'Run and debug view.', path: {
+							dark: 'dark/debug.png',
+							light: 'light/debug.png',
+							hc: 'light/debug.png'
+						},
+					},
+				},
+				{
+					id: 'scmClone',
+					title: localize('gettingStarted.scm.title', "Track your code with Git"),
+					description: localize('gettingStarted.scmSetup.description', "Set up your project to use Git and GitHub integration right inside your editor."),
```

Comment:
"right inside your editor" seems awkward, since they are already inside the editor. Maybe, "set up your project to use Git and GitHub with a click of a button?" to show how easy it is from VS Code?